### PR TITLE
Refactoring bootstrap #2

### PR DESCRIPTION
**Pull request by landsman**
See https://github.com/BurdaMagazinOrg/infinite-theme/pull/41 for more informations.

### CODE REVIEW

File: sass/_variables.scss [hunk 1]
```diff
@@ -414,6 +415,9 @@ $seo-linker__link-list-size: 15px !default;
 $menu-home-animated-logo: true !default;
 $transition-duration: 0.35s !default;
 $transition-easing: ease !default;
+$default-translate-x: 1em;
+$default-translate-y: $default-translate-x;
+$default-translate-z: $default-translate-x;
```
Comment (tjwelde) [inline sass/_variables.scss]: I think we don't need them anymore

File: sass/layouts/_layouts.global.scss [hunk 2]
```diff
@@ -77,3 +77,7 @@ body {
 .visually-hidden {
   display: none;
 }
+
+ol, ul {
+  list-style: none;
+}
```
Comment (tjwelde) [inline sass/layouts/_layouts.global.scss]: we have this already in the reset

File: sass/_variables.scss (hunk 1 above)
Comment (landsman) [reply sass/_variables.scss]: Right, its not used anywhere

File: Gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        interactive: false
+    });
+});
+
+gulp.task('copy-fonts', function ()
+{
+    return gulp
+        .src(["./bower_components/ionicons/fonts/*"])
+        .pipe(gulp.dest('./fonts/ionicons'));
+});
```
Comment (tjwelde) [inline Gulpfile.js]: Why do we copy them to the fonts directory?
AFAICS, we commented them out in sass, but there we go directly to the bower directory.

Comment (tjwelde) [reply Gulpfile.js]: They are also included in this PR in the fonts subdirectoy

File: sass/layouts/_layouts.global.scss (hunk 2 above)
Comment (tjwelde) [reply sass/layouts/_layouts.global.scss]: I tested this and it can be removed.

File: Gulpfile.js (hunk 3 above)
Comment (landsman) [reply Gulpfile.js]: We have two options:
1. using bower_components path for fonts and remove this task
2. keep this task  here and using pretty path

I think that we can remove it from Gulp, files from `/fonts/ionicons` too and using bower path. And we can uncomment import from sass.

**Comment by tjwelde** [comment]
@landsman thanks for your help

**Comment by landsman** [comment]
Can you create new release please?

**Comment by dbosen** [comment]
Before tagging the release, this PR has to be merged: https://github.com/BurdaMagazinOrg/infinite-theme/pull/45 and since the changes are quite big, we need to do a bit more testing first.

**Pull request merged**